### PR TITLE
[plugin.video.telezueri] v0.9.0

### PR DESCRIPTION
**Pull request by goggle**
### Description

This is a video plugin for the local Swiss TV station TeleZüri. You can access the different shows of that TV station through its video library.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by goggle** [comment]
@TravisBuddy Yes I know, **script.module.azmedien** is currently not available in the repository, but it's a pending pull request in the repo-scripts repository, see here: https://github.com/xbmc/repo-scripts/pull/921

File: plugin.video.telezueri/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">TeleZüri media library</summary>
+    <summary lang="de_DE">TeleZüri Mediathek</summary>
+    <description lang="en_GB">Watch all the shows of the local Swiss TV station TeleZüri.</description>
+    <description lang="de_DE">Sehen Sie alle Sendungen des lokalen schweizer Fernsehrsender TeleZüri.</description>
+    <language>de</language>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+    <website>https://www.telezueri.ch</website>
+    <email>seileralex@gmail.com</email>
```
Comment (enen92) [inline plugin.video.telezueri/addon.xml]: please add disclaimer

**Comment by goggle** [comment]
Should I squash these PRs as well?

**Comment by enen92** [comment]
Yes please

**Pull request merged**